docs/network/network_binding_plugins.md

```diff
@@ -189,6 +177,8 @@ Supported types:
 
 - `tap` (from v1.1.1): The domain configuration is set to use an existing
   tap device. It also supports existing `macvtap` devices.
+- `managedTap`: (From 1.4) Manages a bridge with a tap device, without a DHCP server. 
```

Comment:
I'm unsure that this sentence clearly explains the feature.
Please consider the explanation from https://github.com/kubevirt/user-guide/pull/884#discussion_r2048740205 as a base.

Commit message:
> Available since v1
v1.4.

Reply:
It adds a tap interface in case one is not externally provided, the rest is similar to bridge binding - minus the DHCP server.

Reply:
Try this:
> The domain configuration is set to use a tap device. The tap device is wired to the pod interface through a bridge (similar to the core bridge binding) and all are created per need.

Reply:
> No IPAM support is provided with this type.